### ISSUE TITLE
No mention of Tech stack in the video

### ISSUE THREAD

**Issue by william-liebenberg** [wicksipedia/PointBank#18]
The video does not propose any tech stack apart from mentioning "API's"

The tech stack choices can affect the estimation question in the free-lunch form.

**Comment by calumjs** [comment]
@william-liebenberg 
> The video does not propose any tech stack apart from mentioning "API's"
> 
> The tech stack choices can affect the estimation question in the free-lunch form.

Good point, there would be more planning before we could just jump in to developing it...for the free lunch form I think it was assumed that you had free reign to select whichever implementation specifics you want as long as it meets the requirements.
